### PR TITLE
Fix release notes doesn't have newline

### PR DESCRIPTION
**Pull request by elmoiv**
Play Store's release notes return as single line without parsing `<br>` tags.
This commit converts `<br>` tags to `\n` character

### CODE REVIEW

**Comment by larryaasen** [comment]
@elmoiv  This PR did not pass the checks. It cannot be considered until it passes all checks and include adequate unit tests.

File: lib/src/play_store_search_api.dart [hunk 1]
```diff
@@ -99,15 +99,34 @@ class PlayStoreResults {
...
+          ?.querySelector('.DWPxHb');
+      String? innerHtml = rawReleaseNotes!.innerHtml.toString();
+
+      /// Solve the multiline problem with release notes
+      if (releaseNotesSpan.hasMatch(innerHtml)) {
+        releaseNotes =
+            releaseNotesSpan.firstMatch(innerHtml.toString())!.group(1);
+        // Detect default multiline replacement
+        releaseNotes = releaseNotes!.contains('<br><br>')
+            ? releaseNotes.replaceAll('<br><br>', '\n')
```
Comment (Zazo032) [inline lib/src/play_store_search_api.dart:122]: If the release notes have a double line break, why parse it to only one line break?

Comment (elmoiv) [reply lib/src/play_store_search_api.dart]: Thanks for noticing. Fixed it.

**Comment by larryaasen** [comment]
@elmoiv Thanks for this PR. This code change also needs a unit test in `test/play_store_test.dart`.

It failed one TravisCI check in your updated file:
```
$ flutter format -n --set-exit-if-changed ./lib ./test ./example || travis_terminate 1
Changed lib/src/play_store_search_api.dart
Formatted 27 files (1 changed) in 0.68 seconds.
```

File: lib/src/play_store_search_api.dart [hunk 2]
```diff
@@ -99,15 +99,32 @@ class PlayStoreResults {
   /// Returns field releaseNotes from Play Store results. When there are no
   /// release notes, the main app description is used.
   static String? releaseNotes(Document response) {
+    RegExp releaseNotesSpan = RegExp(r'>(.*?)</span>');
```
Comment (larryaasen) [inline lib/src/play_store_search_api.dart:102]: Does this regex need to be compiled on each call to `releaseNotes` or could it be a class const?

File: lib/src/play_store_search_api.dart [hunk 3]
```diff
@@ -99,15 +99,32 @@ class PlayStoreResults {
...
-      final releaseNotes = releaseNotesElement
+
+      String? releaseNotes;
+      
+      Element? rawReleaseNotes = releaseNotesElement
           .querySelector('.PHBdkd')
-          ?.querySelector('.DWPxHb')
-          ?.text;
+          ?.querySelector('.DWPxHb');
+      String? innerHtml = rawReleaseNotes!.innerHtml.toString();
```
Comment (larryaasen) [inline lib/src/play_store_search_api.dart:114]: Using `rawReleaseNotes!` may cause an exception.

**Comment by larryaasen** [comment]
@elmoiv Are you willing to write a unit test to cover this new case? If not, just let me know, and I will write one.

**Comment by elmoiv** [comment]
@larryaasen 
I am sorry for not replying on time. I am busy in the meantime.

I can start on fixing this PR and writing the tests in 2 days.

If you want to write the tests, I will be thankful.

I apologize again for not responding in time.

File: lib/src/play_store_search_api.dart (hunk 2 above)
Comment (elmoiv) [reply lib/src/play_store_search_api.dart]: I moved it as a static variable outside this method

File: lib/src/play_store_search_api.dart (hunk 3 above)
Comment (elmoiv) [reply lib/src/play_store_search_api.dart:115]: Won't the try catch block be able to catch it?

I mean what kind of exception you are exactly thinking of?

**Comment by elmoiv** [comment]
@larryaasen 

I am sorry for that but I am not very familiar with unit testing.

**Comment by larryaasen** [comment]
Thanks for this PR. I will merge it now, and write a unit test.

**Pull request merged**